docs/content/dev/_index.md

```diff
@@ -60,6 +60,96 @@ override it you can set the `PAC_DIRS` environment variable.
   instead of ko. `-c` will only do the pac configuration (ie: creation of
   secrets/ingress etc..)
 
+## Configuring debug logging
+
+pipeline-as-code uses ConfigMap named `pac-config-logging` in the same namespace (`pipelines-as-code` by default) with controllers. To get configmap use the command:
+
```

Comment:
"the ConfigMap"
the same namespace "as the controllers" instead of "with controllers"
"To get the ConfigMap use the following command:"